service/src/test/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCacheTest.java

```diff
@@ -88,9 +91,13 @@ public void testGetWithCacheMiss() {
     public void testGetWithCacheHit() {
```

Comment:
If eval is cached but timestamps for entities are not, then get should return null